src/assets/build_assets/acknowledgements.rs

```diff
@@ -0,0 +1,202 @@
+use std::fs::read_to_string;
+use std::path::Path;
+
+use crate::error::*;
+
+// Sourced from the License section in the README.md
+const PREAMBLE: &str = "Copyright (c) 2018-2021 bat-developers (https://github.com/sharkdp/bat).
```

Comment:
I think this would constitute a good `NOTICE` file btw. If we added this as our own `NOTICE` file, we could `include_str!()` it instead.

Reply:
Sounds good to me. Why `NOTICE` and not `LICENSE`?

Reply:
> Sounds good to me. Why `NOTICE` and not `LICENSE`?

Just to close the loophole that allows others to distribute (possibly modified) binaries of bat without giving credit to https://github.com/sharkdp/bat if they choose the Apache 2.0 license. See [this](https://users.rust-lang.org/t/almost-no-apache-2-0-licensed-rust-project-includes-a-notice-file-how-come/67674/3) for details on what I mean by that.

But calling it a loophole is of course unfair if that scenario is of no concern to you as the founder of this project. I want to be clear that I am fine with if you prefer that we call this file LICENSE and not NOTICE.

Reply:
@sharkdp I have added a NOTICE file now that I `include_str!()`. Let me know if you want it to be called LICENSE instead before I merge.

Reply:
> Just to close the loophole that allows others to distribute (possibly modified) binaries of bat without giving credit to https://github.com/sharkdp/bat if they choose the Apache 2.0 license. See [this](https://users.rust-lang.org/t/almost-no-apache-2-0-licensed-rust-project-includes-a-notice-file-how-come/67674/3) for details on what I mean by that.

Thanks, I was not aware of that.

> @sharkdp I have added a NOTICE file now that I `include_str!()`.

:+1: